src/structlog/contextvars.py

```diff
@@ -30,6 +33,10 @@
 STRUCTLOG_KEY_PREFIX = "structlog_"
 STRUCTLOG_KEY_PREFIX_LEN = len(STRUCTLOG_KEY_PREFIX)
 
+async_calling_stack: contextvars.ContextVar[
```

Comment:
To make it analogue to our other contextvars:

```suggestion
_ASYNC_CALLING_STACK: contextvars.ContextVar[
```

Of course that requires to do the change all over the code base which I'm not gonna do in the web interface here. Feel free to do it locally with F2 in VS Code or something. ;)

Reply:
hah, yeah, just made the search/replace